src/shared/components/event/event.component.ts

```diff
@@ -7,9 +9,27 @@ import { Component, Input, OnInit } from '@angular/core';
 })
 export class EventComponent implements OnInit {
   @Input() event: any;
+  @Input() eventId!: (event: {}) => void;
+
   location = 'assets/icon/mdi_location.svg';
-  date = 'assets/icon/date-icon.svg'
-  constructor() {}
+  date = 'assets/icon/date-icon.svg';
+
+  user: User | null = {
+    email: '',
+    firstName: '',
+    lastName: '',
+    region: '',
+    role: '',
+    organizatorName: '',
+    phone: '',
+    isDeleted: false,
+  };
 
-  ngOnInit() {}
+  constructor(private authService: AuthService) {}
+  deleteEvent(event: {}): void {
+    this.eventId(event);
+  }
+  ngOnInit() {
```

Comment:
ngOnInit should be the first function after ther constructor

Reply:
done